pkg/ingress/admission/api/types.go

```diff
@@ -0,0 +1,19 @@
+package api
+
+import (
+	"k8s.io/kubernetes/pkg/api/unversioned"
+)
+
+// IngressAdmissionConfig is the configuration for the the ingress
+// controller limiter plugin. It changes the behavior of ingress
+//objects to behave better with openshift routes and routers.
+//*NOTE* Disabling this plugin causes ingress objects to behave
+//the same as in upstream kubernetes
```

Comment:
I think we need to say that disabling this has potential security implications in the router when handling ingress objects.